tests/core/core.spec.ts

```diff
@@ -311,8 +319,35 @@ describe("@dataform/core", () => {
       ).to.have.deep.members(expectedResults);
     });
 
+    it("validation_bigquery_fail", () => {
```

Comment:
I think you can give this a slightly more specific name

Reply:
Ah, I replicated from the `"validation_redshift_fail"` from above, as that's used as a parent wrapper for all the redshift specific validation failure tests. Likewise this is the parent wrapper for all bigquery specific validation failure tests (although now there is only one).